mergify_engine/tests/unit/test_github_events.py

```diff
@@ -58,7 +58,7 @@ async def _do_test_event_to_pull_check_run(
         }
     )
     installation = context.Installation(
-        installation_json, mock.Mock(), mock.Mock(), redis_cache
+        installation_json, mock.Mock(), mock.Mock(), redis_cache, mock.ANY
```

Comment:
I guess you mean `mock.Mock()` here. `mock.ANY` is when we don't care about a value/object during an asserting. If we don't care about an object in our code we should use mock.Mock() or mock.MagickMock(). Here it's not a big deal as redis_queue is never used by these tests.

```suggestion
        installation_json, mock.Mock(), mock.Mock(), redis_cache, mock.mock()
```

Reply:
Understood !